### PR TITLE
Change the `bal build` command in Package references page

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Change the `bal build` command to `bal pack` in the `Package references` page.

> Fixes #

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: swan-lake/development-tutorials/source-code-dependencies/package-references.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ visibility = "private"
 
 ### The `org` field
 
-The organization is a logical name used for grouping modules together under a common namespace within a repository. Building a library package with `bal build` and pushing a library package into a repository will fail without an organization name.
+The organization is a logical name used for grouping modules together under a common namespace within a repository. Building a library package with `bal pack` and pushing a library package into a repository will fail without an organization name.
```
Comment (azinneera) [inline swan-lake/development-tutorials/source-code-dependencies/package-references.md:62]:
```suggestion
The organization is a logical name used for grouping modules together under a common namespace within a repository. ```

File: swan-lake/development-tutorials/source-code-dependencies/package-references.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ visibility = "private"
...
 
 Organization names can only contain alphanumerics, underscore, and the maximum length is 256 characters.
```
Comment (azinneera) [inline swan-lake/development-tutorials/source-code-dependencies/package-references.md:64]:
```suggestion
Organization name is mandatory and can only contain alphanumerics, underscore, and the maximum length is 256 characters.
```

**Review by azinneera — APPROVED** [review]

File: swan-lake/development-tutorials/source-code-dependencies/package-references.md [hunk 3]
```diff
@@ -59,9 +59,9 @@ visibility = "private"
 
 ### The `org` field
 
-The organization is a logical name used for grouping modules together under a common namespace within a repository. Building a library package with `bal build` and pushing a library package into a repository will fail without an organization name.
+The organization is a logical name used for grouping modules together under a common namespace within a repository.
```
Comment (anupama-pathirage) [inline swan-lake/development-tutorials/source-code-dependencies/package-references.md:62]: why removing this sentence?

Comment (praneesha) [reply swan-lake/development-tutorials/source-code-dependencies/package-references.md:62]: Removed via https://github.com/ballerina-platform/ballerina-dev-website/pull/8583#discussion_r1461556684. @azinneera

Comment (azinneera) [reply swan-lake/development-tutorials/source-code-dependencies/package-references.md:62]: The compilation itself fails and so we don't need to say about publishing. The next sentence says that its mandatory which gives the information.

**Pull request merged**